src/AltinnCore/ServiceLibrary/Models/ValidateInstantiationStatus.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using AltinnCore.ServiceLibrary.Models;
+
+namespace ServiceLibrary.Models
+{
+    /// <summary>
+    /// A status returned when validating instantiation
+    /// </summary>
+    public class ValidateInstantiationStatus
```

Comment:
A model name should probably not include a verb.. Any suggestions for other names?

Reply:
I see we use Result several places, "ValidationResult", "CodeCompilationResult". Maybe ValidateInstantiationResult? I dont really see how thats any different, but atleast we are consistent.

Reply:
or InstantiationValidationResult . My biggest issue was with the word "validate" so both options would be a good replacement

Reply:
Agreed, fixed.